### PR TITLE
[TASK] Tutorials to Kickstart Extensions

### PR DESCRIPTION
**Pull request by linawolf**
Preview the rendering: https://gitpod.io/#https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/tree/lwolf-make

Releases: main, 11.5

### CODE REVIEW

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+        ..  code-block:: bash
+
+            composer req t3docs/my-test:@dev
+
+    ..  group-tab:: DDEV, Composer
+
+        ..  code-block:: bash
+
+            ddev composer req t3docs/my-test:@dev
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst:173]:
```suggestion
            ddev composer req my-vendor/my-test:@dev
```

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/Index.rst [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.. index::
+   Extension Kickstart; Sitepackage Builder
+.. _extension-sitepackage-builder:
+
+===================
+Sitepackage Builder
+===================
+
+The main incentive of the `Sitepackage Builder <https://www.sitepackagebuilder.com/>`__
+is a quick an easy way to create a new sitepackage, what would be called
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/Index.rst:11]:
```suggestion
is a quick and easy way to create a new sitepackage, what would be called
```

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/Index.rst [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+"theme" in other CMS, for your project. See the
+:doc:`Sitepackage Tutorial <t3sitepackage:Index>` for details on sitepackages.
+
+Kickstart a TYPO3 Extension with the "Sitepackage Builder"
+==========================================================
+
+The `Sitepackage Builder <https://www.sitepackagebuilder.com/>`__ can also be
+used to kickstart a basic extension. You can do so by choosing the
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/Index.rst:19]:
```suggestion
used to kickstart a basic extension. You can do so by choosing
```

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/MinimalExtension.rst [hunk 4]
```diff
@@ -8,6 +8,9 @@
 Minimal extension
 =================
 
+Kickstart a Minimal Extension with the Sitepackage Builder
```
Comment (brotkrueml) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/SitepackageBuilder/MinimalExtension.rst:11]:
```suggestion
Kickstart a minimal extension with the Sitepackage Builder
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+Kickstart a TYPO3 Extension with "Make"
+=======================================
+
+"Make" is a TYPO3 extension provided by B13. It features a quick way to create
+a basic extension scaffold on the console.
+
+1. Install "Make"
+=================
+
+In composer-based TYPO3 installations of v10.4 and above you can install the
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst:19]: I was meaning that this extension is available for v10.4 and above. Maybe we should formulate that in a different way?

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+Kickstart a TYPO3 Extension with "Make"
+=======================================
+
+"`Make <https://github.com/b13/make>`__" is a TYPO3 extension provided by b13. It features a quick way to create
+a basic extension scaffold on the console.
+
+1. Install "Make"
+=================
+
+In composer-based TYPO3 installations of v10.4 and above you can install the
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst:19]:
```suggestion
In Composer-based TYPO3 installations you can install the
```

File: Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+
+====
+Make
+====
+
+Kickstart a TYPO3 Extension with "Make"
+=======================================
+
+"`Make <https://github.com/b13/make>`__" is a TYPO3 extension provided by b13. It features a quick way to create
+a basic extension scaffold on the console.
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Tutorials/Kickstart/Make/Index.rst:14]:
```suggestion
a basic extension scaffold on the console. The extension is available for TYPO3 v10 and above.
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**